scala/scala.bzl

```diff
@@ -487,8 +487,15 @@ def _scala_binary_common(ctx, cjars, rjars):
       compile_jars = depset([outputs.class_jar]),
       transitive_runtime_jars = rjars,
   )
+
+  java_provider = java_common.create_provider(
+      compile_time_jars = scalaattr.compile_jars,
+      runtime_jars = scalaattr.transitive_runtime_jars,
```

Comment:
Is this the way to do it? I'm still honing providers-fu. Shouldn't we use the java provider instead of the scala attr?